From d23ca5754d6a4f89178d7489824b3a3c19486aa2 Mon Sep 17 00:00:00 2001
From: Liang Qi <liang.qi@qt.io>
Date: Thu, 25 Sep 2025 10:04:20 +0200
Subject: [PATCH 263/553] CRA review: plugins/platforms/minimal/*

Only platform backingstore is implemented, not critical.

QUIP: 23
Fixes: QTBUG-QTBUG-135713
Pick-to: 6.8
Change-Id: Ie4e980a364437f0dcaec00c2176d8928a9d6d237
Reviewed-by: Volker Hilsheimer <volker.hilsheimer@qt.io>
(cherry picked from commit 14da3b570fabb97e1a10bb4ac30abf2dff0b7dcf)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/plugins/platforms/minimal/main.cpp                 | 2 +-
 src/plugins/platforms/minimal/qminimalbackingstore.cpp | 2 +-
 src/plugins/platforms/minimal/qminimalbackingstore.h   | 1 +
 src/plugins/platforms/minimal/qminimalintegration.cpp  | 1 +
 src/plugins/platforms/minimal/qminimalintegration.h    | 1 +
 5 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/plugins/platforms/minimal/main.cpp b/src/plugins/platforms/minimal/main.cpp
index 03cb3ca588c..38fe545b127 100644
--- a/src/plugins/platforms/minimal/main.cpp
+++ b/src/plugins/platforms/minimal/main.cpp
@@ -1,6 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
-
+// Qt-Security score:significant reason:default
 
 #include <qpa/qplatformintegrationplugin.h>
 #include "qminimalintegration.h"
diff --git a/src/plugins/platforms/minimal/qminimalbackingstore.cpp b/src/plugins/platforms/minimal/qminimalbackingstore.cpp
index cb432d7ddc9..89db030b146 100644
--- a/src/plugins/platforms/minimal/qminimalbackingstore.cpp
+++ b/src/plugins/platforms/minimal/qminimalbackingstore.cpp
@@ -1,6 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
-
+// Qt-Security score:significant reason:default
 
 #include "qminimalbackingstore.h"
 #include "qminimalintegration.h"
diff --git a/src/plugins/platforms/minimal/qminimalbackingstore.h b/src/plugins/platforms/minimal/qminimalbackingstore.h
index 4bd58fd23d9..b71e50670c9 100644
--- a/src/plugins/platforms/minimal/qminimalbackingstore.h
+++ b/src/plugins/platforms/minimal/qminimalbackingstore.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QBACKINGSTORE_MINIMAL_H
 #define QBACKINGSTORE_MINIMAL_H
diff --git a/src/plugins/platforms/minimal/qminimalintegration.cpp b/src/plugins/platforms/minimal/qminimalintegration.cpp
index 46799674310..a12654734f2 100644
--- a/src/plugins/platforms/minimal/qminimalintegration.cpp
+++ b/src/plugins/platforms/minimal/qminimalintegration.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qminimalintegration.h"
 #include "qminimalbackingstore.h"
diff --git a/src/plugins/platforms/minimal/qminimalintegration.h b/src/plugins/platforms/minimal/qminimalintegration.h
index 6070972b1b8..93c964e3210 100644
--- a/src/plugins/platforms/minimal/qminimalintegration.h
+++ b/src/plugins/platforms/minimal/qminimalintegration.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QPLATFORMINTEGRATION_MINIMAL_H
 #define QPLATFORMINTEGRATION_MINIMAL_H
-- 
2.51.2

